src/test/groovy/com/heroku/sdk/gradle/HerokuPluginTest.groovy

```diff
@@ -44,31 +43,23 @@ class HerokuPluginTest extends Specification {
     def setup() {
         projectDir = temporaryFolder.root
         buildFile = temporaryFolder.newFile('build.gradle')
+        temporaryFolder.newFile("README.md")
```

Comment:
`heroku-deploy` now always includes files relative to the project directory, not the current working dir. This requires us to create this file in the temporary folder since we cannot use the `README.md` from this repo.